src/wollok/lang.wlk

```diff
@@ -141,19 +147,21 @@ class Object {
    *
    * @private
    */
+  // TODO: Move to mirror?
```

Comment:
:+1:  And I like it that moving this code to mirror will not foster the students to use it and check object classes or kinds.

Reply:
That was my thought exactly